api/qsharp/index.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: microsoft.quantum.qsharplibintro
+title: Q# standard library for Microsoft Quantum
+description: Reference documentation for the Q# standard library for Microsoft Quantum
+author: bradben
+ms.author: bradben
+ms.date: 03/04/2020
+ms.topic: landing-page
+---
+
+
+# Q# libraries #
+
+Q# is supported by a range of different useful operations, functions, and user-defined types that comprise the Q# *standard library*.
```

Comment:
Same comment here as well. Mentioning the other three libraries as well as the standard library would be helpful, I think.